src/components/Board.tsx

```diff
@@ -12,71 +12,80 @@
 //      * Daniel Schilling
 //
 
+import {useState} from 'react';
 import Square from './Square.tsx';
 
 /**
  * Board of the connect-four game.
  */
 function Board() {
-    function handleClick() {
-        console.log('Clicked!');
+    const [value, setValue] = useState<('red' | 'blue' | null)[]>(
+        Array.from({length: 43}, () => null)
+    );
```

Comment:
`value` is not a good name, since it doesn't describe what this is good for (and unlike in `<square>`, where there is realy only one thing it could be, `<board>` will probably need more pieces of state in the future). Maybe name it something like `squares` or `tokens`?